app/views/users/new.html.erb

```diff
@@ -0,0 +1,41 @@
+<% provide :title, t("user.signup") %>
+<h1><%= t "user.signup" %></h1>
+
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_with model: @user, url: signup_path, local: true do |f| %>
+      <%= render "shared/error_messages", object: f.object %>
+
+      <%= f.label :role, t("user.role") %>
+      <%= f.radio_button :role, "1", checked: true %><%= t("user.employeer")%>
+      <%= f.radio_button :role, "2" %><%= t("user.candidate")%><br>
+
+      <%= f.label :name, t("user.name") %>
```

Comment:
chỗ này em i18n bằng cách này nha
https://guides.rubyonrails.org/i18n.html#translations-for-active-record-models